index.css

```diff
@@ -192,9 +189,19 @@ h2{
         font-size: 40px;
         align-self: flex-end;
     }
+    
+    #projects{
+        margin-top: 8em;
+    }
+    .project{
+        margin-bottom: 8.5em;
+    }
+    .info{
```

Comment:
thanks @maguzzz ,
 but  this line of code is disrupting the text alignment,
![Screenshot (27)](https://user-images.githubusercontent.com/111265239/212581033-5cdf2f07-348f-42a5-b44f-35657b70b3e1.png)